### ISSUE TITLE
Checkbox and radio not visible in white-on-black high contrast mode

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#236]
See #233 for details.

![screenshot 10](https://user-images.githubusercontent.com/38065/41736394-2ad83536-7541-11e8-8483-c0a7affe774d.png)

![screenshot 14](https://user-images.githubusercontent.com/38065/41736466-730b88c6-7541-11e8-8430-dffb07fb42fe.png)
